### DISCUSSION TITLE
Bot mouseover floods chats with whisper reminder of commands

### DISCUSSION THREAD

**Discussion by tmscaf** [mod-playerbots/mod-playerbots#202] [General]
Is anyone else experiencing a situation where when you mouse over any bot in your party, it will immediately whisper the list of commands to you? Where as I appreciate a reminder of what commands control a bot, it's floods the chat windows, especially when having a full party. Is this normal behavior?

**Comment by liyunfan1223** [comment]
Similar to https://github.com/liyunfan1223/mod-playerbots/issues/218. It's likely caused by addons.
